### PR TITLE
Add reactive-streams to non-standard.json

### PR DESCRIPTION
**Pull request by ktoso**
To include http://www.reactive-streams.org/ so we're able to find libraries which implement them

### CODE REVIEW

File: non-standard.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "org.scala-lang scala-reflect":                    "version",
   "org.scala-lang scala-xml":                        "version",
   "org.scala-lang scalap":                           "version",
+  "org.reactivestreams reactive-streams":            "pom",
```
Comment (MasseGuillaume) [inline non-standard.json]: http://search.maven.org/#artifactdetails%7Corg.reactivestreams%7Creactive-streams%7C1.0.0%7Cjar

ibid: https://github.com/scalacenter/scaladex-contrib/pull/15

`"java"`

Comment (ktoso) [inline non-standard.json]: I'll attempt implementing this, may the gods may mercy 😉

Comment (ktoso) [inline non-standard.json]: Initial PR: https://github.com/scalacenter/scaladex/pull/305

**Comment by MasseGuillaume** [comment]
https://github.com/scalacenter/scaladex-contrib/pull/18

**Pull request closed without merging**